src/main/java/frc/robot/lib/states/Arm.java

```diff
@@ -37,4 +74,62 @@ public void moveArmPosition(double firstStage, double secondStage, double wrist)
         RobotMap.armSecondStage.set(TalonFXControlMode.Position, secondStage);
         RobotMap.wrist.set(TalonFXControlMode.Position, wrist);
     }
+
+    /* getClosedLoopError(): Gets the closed-loop error. The units depend on which control mode is in use. 
+    If closed-loop is seeking a target sensor position, closed-loop error is the difference 
+    between target and current sensor value (in sensor units. 
+    Example 4096 units per rotation for CTRE Mag Encoder). */
+
+    public boolean getArrived(double allowance) {
```

Comment:
This function would be useful to have on a per-motor basis, instead of it being specific to the arm subsystem.